lib/improver/blending/weighted_blend.py

```diff
@@ -125,17 +238,31 @@ def conform_metadata(
             ndim = cube.coord_dims("time")
             cube.add_aux_coord(forecast_period, data_dims=ndim)
 
+    # update blended cube attributes
+    if "title" not in cube.attributes.keys():
+        cube.attributes["title"] = "IMPROVER Model Forecast"
+
+    if ("model" in coord) and any(
+            "mosg__" in key for key in cube_orig.attributes.keys()):
+        cube.attributes["mosg__model_configuration"] = "blend"
```

Comment:
Is this ok to set the mosg__model_configuration to "blend"? Isn't it fairly controlled as it is a piece of StaGE metadata, or has this been agreed?

Reply:
This was agreed / stated in Caroline's report for IMPRO-429, and has been added to the confluence page describing the current state of IMPROVER blended metadata.